.gitmodules

```diff
@@ -1,7 +1,3 @@
-[submodule "external/guiunit"]
-    path = external/guiunit
-    url = ../../mono/guiunit.git
-    branch = main
```

Comment:
Not sure about this.

Reply:
should be identical to `d16-8` since we have not touched this in `xcode12.x`